api/src/pcapi/routes/serialization/statistics_serialize.py

```diff
@@ -9,13 +11,17 @@ class StatisticsQueryModel(BaseModel):
 class StatisticsModel(BaseModel, YearlyAggregatedRevenueModel):
     @classmethod
     def from_query(cls, income_by_year: dict) -> "StatisticsModel":
+        current_year = datetime.utcnow().year
```

Comment:
Détail : Pas besoin de l'heure 
```suggestion
        current_year = date.today().year
```